src/components/AddUser/AddUser.tsx

```diff
@@ -0,0 +1,115 @@
+import { useState, FormEvent, FC } from 'react';
+import { Todo } from '../../types/types';
+import { getUserById } from '../../utils/getUserByID';
+import usersFromServer from '../../api/users';
```

Comment:
[CODE STYLE] - Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components. You're importing `usersFromServer` directly in the `AddUser` component. It would be better to import this data in the parent component and then pass it as a prop.